### ISSUE TITLE
Add bidi support

### ISSUE THREAD

**Issue by ahangarha** [pixelfed/pixelfed#1349]
Hi

Please add bidi (bidirectional) support to pixelfed. So the text direction (LTR/RTL) would be determined by the browser automatically.

I have experience of doing this for few other projects (Plume, FluxBB,...) so you might assign this task to me.

**Comment by dansup** [comment]
@ahangarha Hello, 

We support ```<bdi>``` in [usernames](https://github.com/pixelfed/pixelfed/search?q=bdi%3E&unscoped_q=bdi%3E) but not in captions or comments. We will add support in the next release!

**Comment by ahangarha** [comment]
Why closed? Why reopened?
Does this mean anyone is going to work on this?

**Comment by dansup** [comment]
@ahangarha We added ```<bdi>``` support! 

https://github.com/pixelfed/pixelfed/search?q=%3Cbdi%3E&unscoped_q=%3Cbdi%3E

**Comment by ahangarha** [comment]
How does it help? This is applied on very few things and still the body of posts, are not affected

**Comment by ahangarha** [comment]
I checked again the impact of your change. It doesn't do what should happen. look at this post for instance: https://pixelfed.social/p/Ahangarha/168307740094107648

![image](https://user-images.githubusercontent.com/11241315/82374770-c5755780-9a34-11ea-9146-8fcf52593707.png)

The text is LTR regardless of the fact that the second line is in Persian and should be RTL.

Putting bdi to whole text is not helping when the description is mixed with RTL and LTR text. I also think it is a mistake to keep different paragraphs in `span` tag. they should be kept in `p`. Then, `p` tags can have `dir="auto"` and the issue will be resolved.
